meta-boxes.php

```diff
@@ -1,20 +1,139 @@
 <?php
+/**
+ * File containing meta box callback functions
+ */
 
-function nprstory_publish_meta_box($post) {
-	$helper_text = 'Push this story to NPR:';
-	$is_disabled = ($post->post_status != 'publish');
-	if ($is_disabled) {
-		$helper_text = 'Publish this story in order to push it to NPR.';
+/**
+ * Output the NPR Story API publishing options metabox for the edit page admin interface
+ *
+ * @param WP_Post $post the WordPress post object.
+ * @see nprstory_save_send_to_api
+ * @see nprstory_save_send_to_org
+ * @see nprstory_save_send_to_one
+ * @see nprstory_save_nprone_featured
+ *
+ * @todo When there is better browser support for input type="datetime-local", replace the jQuery UI and weird forms with the html5 solution. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local
+ */
+function nprstory_publish_meta_box( $post ) {
+	$helper_text = __( 'Push this story to NPR:', 'nprapi' );
```

Comment:
I think this is supposed to be "Send to NPR API"

Reply:
In the current version of the plugin, `$helper_text` is displayed atop the meta box, but it's not used at all in this pull request, so I'll remove it.

Here's how it was formerly used:

https://github.com/npr/nprapi-wordpress/blob/6cdfea4528b8c6eabcc9b5b2576b2f564dda9bee/meta-boxes.php#L14

<img width="297" alt="screen shot 2018-04-16 at 7 47 01 pm" src="https://user-images.githubusercontent.com/1754187/38840714-00c7d4a6-41af-11e8-855f-1f19057d11b1.png">


And here's where it isn't being used at all, now:

<img width="297" alt="screen shot 2018-04-16 at 7 43 17 pm" src="https://user-images.githubusercontent.com/1754187/38840595-7effa516-41ae-11e8-84ca-ef4facf72e1c.png">